lib/tasks/fix_missing_organisations.rake

```diff
@@ -0,0 +1,20 @@
+#rake goodcity:fix_missing_organisations
+
+namespace :goodcity do
+  task fix_missing_organisations: :environment do
+    count = 0
+    log = Goodcity::RakeLogger.new("fix_missing_organisations")
+    #get all the gc orders without organisation and not draft draft orders
+    Order.where("orders.state <> ? and orders.organisation_id IS NULL and orders.created_by_id IS NOT NULL and code ILIKE ?","draft", "GC-%").find_each do |order|
+      creator_organisations = order.created_by&.organisations
+      next unless creator_organisations
+      if order.update(organisation_id: creator_organisations.first.id)
```

Comment:
Could you please verify once if there is any callback which can cause an issue as we are using an update? If not then great. :)

Reply:
I have purposely used update so that callbacks get fired and the jobs which are intended, they get fired while updating the organisation for order. I have checked push_changes callback gets fired, which will run a job for updating the order @namrataukirde .